### DISCUSSION TITLE
[A-L] (2024/25) Foglio 1 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Rurik-D** [sapienzastudentsnetwork/algebra#466] [Esercizi - Viaggi/Cherubini]
![image](https://github.com/user-attachments/assets/21610f0a-2df9-4810-84d3-ad823271b46a)

**Comment by IonutCicio** [comment]
**riflessività**: dato $A \in U$ basta prendere la funzione $f : A \to A$ identità su $A$. La funzione identità è biettiva, quindi $\forall A \in U \ \ \ A \sim A$ 
**simmetria**: dati $A, B \in U$, se $A \sim B \implies \exists f : A \to B$ biettiva. Ma se $f$ è biiettiva allora è invertibile, e la sua inversa $f' : B \to A$ è a sua volta biettiva, quindi $\forall A, B \in U \ \ \ A \sim B \implies B \sim A$
**transitività**: dati $A, B, C \in U$, se $A \sim B \land B \sim C \implies \exists \ f: A \to B \land g: B \to C$ entambe biettive, ma dato che la composizione di due funzioni biettive è a sua volta biettiva, si ha che $g \circ f : A \to C$ è a sua volta biettiva, quindi $A \sim C$, quindi $\forall A, B, C \in U \ \ \ A \sim B \land B \sim C \implies A \sim C$
